src/ai/backend/storage/config.py

```diff
@@ -30,6 +30,7 @@
                     t.Key("max-upload-size", default="100g"): tx.BinarySize,
                     t.Key("secret"): t.String,  # used to generate JWT tokens
                     t.Key("session-expire"): tx.TimeDuration,
+                    t.Key("hide_vfolder_usage", default=True): t.ToBool,
```

Comment:
1. the name of other options use `-` instead of `_`.

2. To meet the requirement, depending on the settings it will determine which information to expose: 
- return `percentage` only
- return `percentage`, `usage` and `total`.
- return nothing.

@adrysn , right?